stock_demand_estimate/wizards/stock_demand_estimate_wizard_view.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+<odoo>
+
+        <record model="ir.ui.view"
+                id="view_stock_demand_estimate_sheet_form">
+            <field name="name">stock.demand.estimate.sheet.form</field>
+            <field name="model">stock.demand.estimate.sheet</field>
+            <field name="arch" type="xml">
+                <form string="Stock Demand Estimate Sheet">
+                    <group>
+                        <group name="dates">
+                            <label for="date_start" string="Timesheet Period"/>
```

Comment:
IMO string is wrong here.

Reply:
it is there for the whole block

Reply:
I should have expressed myself clearly enough. :smile:  
The word timesheet doesn't make sense : https://dictionary.cambridge.org/dictionary/english/timesheet
Leaving only 'Period' would make more sense as it is in the view below. :wink: